website/src/components/Zoom/index.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {useColorMode} from '@docusaurus/theme-common';
+import BasicZoom from 'react-medium-image-zoom';
```

Comment:
this seems to load quite a few extra deps, see lockfile

what about using an existing docusaurus plugin instead?

https://github.com/flexanalytics/plugin-image-zoom
https://github.com/gabrielcsapo/docusaurus-plugin-image-zoom

this could apply to all site images automatically and also be nice to dogfood community plugins